docs/guide/object-road.md

```diff
@@ -840,7 +840,7 @@ RoadIntersection {
          MFFloat              startRoadsLength               [5]
          SFInt32              startRoadsNumberOfLanes        2
          MFString             startRoadsStartLine            [ "textures/road_line_dashed.png",
-                                                               "textures/road_line_triangle.png" ]
+                                                                     "textures/road_line_triangle.png" ]
```

Comment:
It seems wrong that these additional spaces are added.
We should probably add an issue about this problem with the object regeneration script if it was added automatically.

Reply:
Yes I used the script

Reply:
So then we should open an issue to try to fix the script.